collie/metrics.py

```diff
@@ -277,7 +277,7 @@ def auc(targets: csr_matrix,
 
 
 def evaluate_in_batches(
-    metric_list: Iterable[Callable],
+    metric_list: Iterable[Callable[..., Any]],
```

Comment:
In this case, we actually do know the expected inputs/outputs of the metric functions, so we can have this be: 

```suggestion
    metric_list: Iterable[Callable[[csr_matrix, Union[np.array, torch.tensor], Union[np.array, torch.tensor], Optional[int]], float]],
```

Reply:
oof, thank you!